tests/commands/train_test.py

```diff
@@ -28,6 +30,28 @@
 SEQUENCE_TAGGING_SHARDS_PATH = str(AllenNlpTestCase.FIXTURES_ROOT / "data" / "shards" / "*")
 
 
+@BatchCallback.register("training_data_logger")
+class TrainingDataLoggerBatchCallback(BatchCallback):
+    def __call__(
+        self,
+        trainer: "GradientDescentTrainer",
+        batch_inputs: List[List[TensorDict]],
+        batch_outputs: List[Dict[str, Any]],
+        epoch: int,
+        batch_number: int,
+        is_training: bool,
+        is_master: bool,
+    ) -> None:
+        if is_training:
+            logger = logging.getLogger(__name__)
+            for batch in batch_inputs:
+                for metadata in batch["metadata"]:
+                    logger.info(f"First word from training data: '{metadata['words'][0]}'")
+
+    def in_worker(self, *args, **kwargs) -> None:
```

Comment:
What's this doing?  I don't see it called anywhere.

Reply:
Sorry, that was a leftover from an earlier iteration. I removed it.